src/abhijitMJun20/Bankingsystem.java

```diff
@@ -0,0 +1,52 @@
+package abhijitMJun20;
+
+public class Bankingsystem {
+	int accountbalance=1000;
+	static int credittransactionsummary=0,debittransactionsummary=0,printbalancesummary=0;
+	int creditcount=0,debitcount=0,printBalancecount=0;
+	
+	void debitAmount(int debitamount) { 
+		accountbalance=accountbalance-debitamount;
```

Comment:
You should write the condition to verify the insufficient balance.

Reply:
Done